setup.py

```diff
@@ -61,6 +61,7 @@
         'requests',
         'setuptools',
         'tarjan',
+        'plone.app.jquery >= 1.9.1',
```

Comment:
I agree with your point.  

I'll do this instead:
Remove the requirement of plone.app.jquery >= 1.9.1
Modify manage.js to raise an error if jQuery <= 1.7 is being used.